### PR TITLE
Ex webrtc integration test

### PR DESCRIPTION
**Pull request by roznawsk**

### CODE REVIEW

File: ex_webrtc/integration_test/test_videoroom/test/integration/basic_test.exs [hunk 1]
```diff
@@ -56,15 +56,15 @@ defmodule TestVideoroom.Integration.BasicTest do
           {:after_warmup, browsers} ->
             Enum.each(browsers, fn {browser_id, stats_list} ->
               Enum.each(stats_list, fn stats ->
-                assert length(stats) == browser_id
```
Comment (roznawsk) [inline ex_webrtc/integration_test/test_videoroom/test/integration/basic_test.exs:59]: fyi: Now each track is a separate stream, so the expected number of streams is double the number of peers

File: ex_webrtc/test/ex_webrtc/track_sender_test.exs [hunk 2]
```diff
@@ -172,7 +172,7 @@ defmodule Membrane.RTC.Engine.ExWebRTC.TrackSenderTest do
     pipeline = build_video_pipeline(track, {nil, &Utils.generator/2}, 3)
 
     new_bitrates = %{high: 1420, medium: 689}
-    Pipeline.message_child(pipeline, :track_sender, {:variant_bitrates, new_bitrates})
+    Pipeline.notify_child(pipeline, :track_sender, {:variant_bitrates, new_bitrates})
```
Comment (roznawsk) [inline ex_webrtc/test/ex_webrtc/track_sender_test.exs:175]: fyi: deprecated function in Membrane

File: ex_webrtc/lib/ex_webrtc_endpoint.ex [hunk 3]
```diff
@@ -459,20 +447,47 @@ defmodule Membrane.RTC.Engine.Endpoint.ExWebRTC do
...
+      outbound_tracks = Map.put(state.outbound_tracks, track_id, track)
+
+      {actions, %{state | outbound_tracks: outbound_tracks}}
+    catch
+      :exit, _err ->
+        Process.send_after(self(), {:subscribe_result, subscribe_ref, :ok}, 100)
+
+        {[], state}
+    end
+  end
```
Comment (Karolk99) [inline ex_webrtc/lib/ex_webrtc_endpoint.ex:480]: i think this block needs some comment why it's written this way

File: ex_webrtc/test/ex_webrtc/peer_connection_handler_test.exs [hunk 4]
```diff
@@ -12,6 +12,7 @@ defmodule Membrane.RTC.Engine.Endpoint.ExWebRTC.PeerConnectionHandlerTest do
   alias Membrane.RTC.Engine.Track
 
   @endpoint_id "ex_webrtc_endpoint"
+  @track_metadata %{"server" => %{"displayName" => "mrwebrtc"}, "peer" => %{}}
```
Comment (Karolk99) [inline ex_webrtc/test/ex_webrtc/peer_connection_handler_test.exs:15]: 🏎️

**Review by Karolk99 — CHANGES_REQUESTED** [review]

File: ex_webrtc/lib/ex_webrtc_endpoint.ex [hunk 5]
```diff
@@ -117,7 +126,7 @@ defmodule Membrane.RTC.Engine.Endpoint.ExWebRTC do
       |> child({:track_receiver, track_id}, %TrackReceiver{
         track: track.engine_track,
         initial_target_variant: :h,
-        keyframe_request_interval: Membrane.Time.seconds(5)
+        keyframe_request_interval: Membrane.Time.seconds(3)
```
[the remark this answers is not in the document]
Comment (roznawsk) [reply ex_webrtc/lib/ex_webrtc_endpoint.ex:129]: 👍🏽

File: ex_webrtc/lib/ex_webrtc/peer_connection_handler.ex [hunk 6]
```diff
@@ -158,13 +200,38 @@ defmodule Membrane.RTC.Engine.Endpoint.ExWebRTC.PeerConnectionHandler do
     {[], state}
   end
 
+  @impl true
+  def handle_parent_notification({:tracks_removed, track_ids}, _ctx, state) do
```
[the remark this answers is not in the document]
Comment (roznawsk) [reply ex_webrtc/lib/ex_webrtc/peer_connection_handler.ex:204]: There is a task for this

File: ex_webrtc/lib/ex_webrtc_endpoint.ex [hunk 7]
```diff
@@ -459,20 +478,36 @@ defmodule Membrane.RTC.Engine.Endpoint.ExWebRTC do
...
+        Process.send_after(self(), {:subscribe_result, subscribe_ref, :ok}, 100)
 
-    outbound_tracks =
-      case result do
-        :ok -> Map.put(state.outbound_tracks, track_id, track)
-        # TODO: build remove actions and notify handler
-        :ignored -> Map.delete(state.outbound_tracks, track_id)
-      end
+        {[], state}
+    end
```
[the remark this answers is not in the document]
Comment (roznawsk) [reply ex_webrtc/lib/ex_webrtc_endpoint.ex:497]: This will be done in separate PR, as discussed.

**Review by Karolk99 — APPROVED** [review]

**Comment by roznawsk** [comment]
Dorime 🙏🏽

**Pull request merged**